build/devices/esp32/targets/m5stack_core2/setup-target.js

```diff
@@ -70,7 +72,13 @@ export default function (done) {
 	};
 
 	// power
-	globalThis.power = new Power();
+  const powerICID = new SMBus({
```

Comment:
While this code does work, it would be best to immediately close the SMBus instance rather than waiting for it to be garbage collected. I suggest the following:

```js
  const s = new SMBus({
     ...INTERNAL_I2C,
     address: 0x34,
   })
  const powerICID = s.readByte(0x03)
  s.close()
```

Reply:
@meganetaaan - What do you think about this change? If you can make the adjustment, then we should be able to merge this. Thank you!

Reply:
Much better! I confirmed that the fix works then pushed.

Reply:
Great, thanks for making the change.